src/constants.cr

```diff
@@ -22,6 +22,8 @@ module PlaceOS::Api
   INFLUX_HOST    = ENV["INFLUX_HOST"]?
   INFLUX_ORG     = ENV["INFLUX_ORG"]? || "placeos"
 
+  NEW_RELIC_KEY = ENV["NEW_RELIC_KEY"]?
```

Comment:
```suggestion
  OPENTELEMETRY_PROVIDER_KEY = ENV["OPENTELEMETRY_PROVIDER_KEY"]?
```

Reply:
I think we should support

- `OPENTELEMETRY_PROVIDER_KEY`
- `ELASTIC_APM_API_KEY`
- `NEW_RELIC_KEY`